dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -104,11 +103,11 @@ protected String parseCommandText(String commandText) {
         return commandText;
     }
 
-    public final PreparedStatement createStatementWithBoundFixtureSymbols(
+    @Override
+    public final DbStatement createStatementWithBoundFixtureSymbols(
             TestHost testHost, String commandText) throws SQLException {
         String command = Options.isBindSymbols() ? parseCommandText(commandText) : commandText;
-        PreparedStatement cs = getConnection().prepareStatement(
-                command);
+        PreparedStatement cs = getConnection().prepareStatement(command);
```

Comment:
`cs` perhaps stands for `callable statement`. We may rename that to `statement` here (and not only here). It's indeed more an exception than a rule to get a callable statement here.